server/models/user.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+// Placeholder User schema
+const UserSchema = new Schema({
+  firstname: { type: String, required: true },
+  lastname: { type: String, required: true },
+  email: { type: String, required: true },
+  password: { type: String, required: true },
+});
```

Comment:
Looks like a good starting point, I'd probably make email unique 
Are we using a linter / prettier on this project yet?